### ISSUE TITLE
Documentation does not match latest release

### ISSUE THREAD

**Issue by mmahrouss** [nolar/kopf#589]
## Long story short

I was trying to use `kopf.on.update` but the `field` parameter was not there because apparently it is not in the latest release

The documentation defaults to `latest` which does not allign with latest release 0.28 

The `stable` documentation seem up to date on the latest release
* Kopf version: 0.28
* Python version: 3.6
* OS/platform: linux

Finally, thank you and I really appreciate your work to bring this to life

**Comment by nolar** [comment]
Yep, good point. I've changed the default documentation from "latest" to "stable", so that people do not get confused anymore. 

It was not a problem when the codebase was small and the development was fast and full-time, but now, there are gaps between a feature is merged and it is released.

Thanks for pointing that out. 

If you want to try that specific feature, use can `pip install git+https://github.com/nolar/kopf.git@master` — but beware of the risks of general stability, as usually with pre-release/release-candidate software.
